wayback/_utils.py

```diff
@@ -1,6 +1,7 @@
 from collections import defaultdict
 from contextlib import contextmanager
 from datetime import date, datetime, timezone
+import logging
```

Comment:
Can you please also create a logger so people know *where* this warning is coming from if they hit it? e.g:

```py
logger = logging.getLogger(__name__)
```

Reply:
Nice catch, logger is configured now.